src/main/resources/templates/entry.html

```diff
@@ -47,6 +47,12 @@ <h1 class="entry-header" th:text="${record.primaryKey}">Entry Name Header</h1>
     </div>
 </div>
 
+<div class="row">
+    <div class="small-12 large-6 columns">
+        <p>View all versions of this record <a th:href="${@uk.gov.register.presentation.HtmlViewSupport@extractLinkFromLinkHeader(#ctx.httpServletResponse.getHeader('Link'))}">here</a></p>
```

Comment:
extracting the link from the link header feels fragile (because we might later add another link rel in another header) and wrong (because the view should be able to provide the link directly)